app/assets/scripts/components/new/donut-chart.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
```

Comment:
Can the `donut-chart` component just go in `scripts/components/` - I don't see a reason for this to be under `new` ?

Reply:
I'm using the `new` directory to keep track of the things I added, I'll move these components appropriately later on